meos/src/point/tpoint_analytics.c

```diff
@@ -813,7 +813,7 @@ tsequence_simplify_min_dist(const TSequence *seq, double dist)
         last = true;
     }
   }
-  if (! last)
+  if (k > 1 && ! last)
```

Comment:
I think this should be:
_if (seq->count > 1 && ! last)_
Indeed, we can have k == 1 if we only maintain the start and end instants (ex. 2 instant sequence where start and end are closer than min_dist